roles/iam-user/tasks/create_users.yml

```diff
@@ -0,0 +1,26 @@
+
+---
+- name: Create IAM users
+  iam:
+    state: present
+    iam_type: user
+    name: "{{ item.name }}"
+    groups: "{{ item.groups }}"
```

Comment:
there is no link between this and the group that is created. the logic could use the iam_user_users variable to provide details for user and group creation